### PR TITLE
New Lessvar for Plone Toolbar Submenu - fix typo

### PR DESCRIPTION
**Pull request by jensens**
upgrades for toolbar enhancements

### CODE REVIEW

File: plone/app/upgrade/v50/final.py [hunk 1]
```diff
@@ -92,6 +95,27 @@ def to502(context):
...
+        prefix='plone'
+    )
+    _marker = list()
+    value = rreg.lessvariables.get('plone-toolbar-font-secundary', _marker)
+    if (value == _marker):
+        return
+    if 'plone-toolbar-font-secondary' in rreg.lessvariables:
+        logger.warn(
+            'Try to migrate registry value "plone-toolbar-font-secundary" to '
+            '"plone-toolbar-font-secundary", but latter already exists. '
```
Comment (mauritsvanrees) [inline plone/app/upgrade/v50/final.py]: In the log warning you are migrating secundary to secundary, so typo to typo, if I see it correctly.

File: plone/app/upgrade/v50/final.py [hunk 2]
```diff
@@ -92,6 +95,27 @@ def to502(context):
...
+            'Migration to fix the typo is not executed.'
+        )
+        rreg.lessvariables['plone-toolbar-font-secondary'] = value
+        del rreg.lessvariables['plone-toolbar-font-secundary']
```
Comment (mauritsvanrees) [inline plone/app/upgrade/v50/final.py]: The indentation looks wrong. Bit hard to see with my comment in between. :-)
It is now:

```
if secondary without typo already exists:
    Warn that migration is not executed.
    Execute migration anyway.
```

File: plone/app/upgrade/v50/final.py (hunk 1 above)
Comment (jensens) [inline plone/app/upgrade/v50/final.py]: lol, indeed, I'll fix

File: plone/app/upgrade/v50/final.py (hunk 2 above)
Comment (jensens) [inline plone/app/upgrade/v50/final.py]: sorry, I dont get what you mean?

Comment (mauritsvanrees) [inline plone/app/upgrade/v50/final.py]: If there is both secundary and secondary, the code warns that migration will not be executed and then in de next line the migration is executed anyway.

I am expecting this, in pseudo code:

```
If secondary already in registry:
    Warn that migration will not be done.
    Return.
Do migration
```

**Comment by thet** [comment]
LGTM

**Comment by thet** [comment]
Upgrade step has to target Plone 5.1

**Pull request merged**